zookeeper-server/src/main/java/org/apache/zookeeper/ClientCnxn.java

```diff
@@ -1268,11 +1256,7 @@ public void run() {
                         } else if (e instanceof SocketException) {
                             LOG.info("Socket error occurred: {}: {}", serverAddress, e.getMessage());
                         } else {
-                            LOG.warn("Session 0x{} for server {}, unexpected error{}",
-                                            Long.toHexString(getSessionId()),
-                                            serverAddress,
-                                            RETRY_CONN_MSG,
-                                            e);
+                            LOG.warn("Session 0x{} for server {}, unexpected error{}", Long.toHexString(getSessionId()), serverAddress, RETRY_CONN_MSG, e);
```

Comment:
This line after reformatting looks awfully long. Is it possible to teach check style so it insert line breaks when a single line exceeds certain preconfigured width?

Reply:
It is possible that we enable `LineBreak` rule on zookeeper-server that forbid every line exceed a preconfigure value. See ZOOKEEPER-3508 for it.

We can tell an auto-formatter to break a line but it would introduce awful breaks as Enrico request to change.

Reply:
In fact I locally turn on `LineLength` and see that there is about 200 files break the rules, a pass to take care of it is ok for this pr. But not all of them have a reasonable break down strategy to apply on. Some of them are just because too much indent level so break line length rule.